rfcs/0074-community-coordination-hub.md

```diff
@@ -0,0 +1,153 @@
+- Organizational Structure, that enacts necesary degrees of _Division of Labor_ 
+  and _Coordination_.
+- Organizational Culture, that helps to overcome the complexity bias. ([KISS](https://en.wikipedia.org/wiki/KISS_principle))
+
+# Detailed design
+[design]: #detailed-design
+
+A repository shall exist, named `NixOS/community`, which will act as central hub
+for coordinating the community's efforts.
+
+It shall comprehend, consolidate and showcase aspects of:
+
+- Communication
+  - Channels
+  - Modes & Policies
+- Governance
+  - Processes
+  - Structure
+- Onboarding
+  - Code Contributions
+  - Community Engagement
+
+It shall omit any technical subject about nix or code in general, other than
+repository tooling.
+
+To implement, myself, and anyone interested, will review https://github.com/kubernetes/community
+to the smallest details.
+
+Based on this review, we'll create a clean new repository at
+`NixOS/community`, with what we considered was worth porting from
+[Kubernetes' community portal](https://github.com/kubernetes/community).
+
+As a general guideline for this review, we'd acknowledge the success
+and impressive dynamics of the Kubernetes ecosystem.
+
+## Brainstorm
+[brainstorm]: #brainstorm
+
+This section exposes loose ideas we might put into `NixOS/community` collected during
+this RFC's review process.
+
+- [Community Event Calender](https://github.com/kubernetes/community/tree/master/events)
+- Explain and Support an Efficient Code Review Process.
+
+# User Stories
+[user-stories]: #user-stories
+
+If this RFC is implemented ...
+
+## As a newcomer
+As a newcomer, I'd be able to quickly find my way around, and start beeing
+useful before beeing turned down by inconsistent and spread-out onboarding.
+
+## As a veteran
+As a veteran, I'd be able to promote some ideas, that I care about,
+towards completion while suffering from less friction and overly complex
+coordination. I can leverage my experience and skills more effectively
+for the benefit of the community.
+
+## As a mentor
+As a mentor, I can inform my efforts with a clear and consistent reference.
+It helps me to more effectively maintaining those high quality human 
+interactions that the NixOS community is known for.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+There _can_ be a better solution to achieve _optimum levels of empowerment_
+for our community. We _could just **wait**_ for them to unveil themselves.
+
+# Alternatives
+[alternatives]: #alternatives
+
+- [Benevolent Dictatorship](https://discourse.nixos.org/t/what-would-you-do-if-you-were-the-bdfl-of-nix/6949)
+- Ephemeral Conversations (Discourse)
+
+The only current viable alternative for _coordination of resources and effort_
+are ephemeral conversations in Discourse. While they lend meaning to the 
+individual's interpretations and views onto the ecosystem, they lack important
+features such as:
+
+- Structure
+- Consolidation
+- Ability to act as _Reference_
```

Comment:
1. Please explain why you consider Discourse inferior on these points.  (I can imagine *vaguely* how a GitHub repository might be superior in the last two regards, but I don't want to need to imagine.)

2. In what way are Discourse discussions problematically less structured than GitHub discussions?  If anything, I would think Discourse discussions slightly more structured, as the one difference that comes to mind is that Discourse discussions have a threaded structure and GitHub discussions have a flat structure.

Reply:
> Discourse inferior on these points

I want to avoid spending lines on why Discourse is bad. It isn't. It's complementary. But you are quite right in so far that some exemplification might improve the argument.

> structured than GitHub discussions

`kubernetes/community` - in my interpretation - is not a place for conversations. Rather for _reference_ and _organization_. The discussions take place in other spaces. We might want to make that clearer in order to avoid misinterpretations. Thanks for bringing up this interpretation!

Reply:
How does it differ from Nixpkgs manual, then? It has some issues but it is in a GitHub repository and already documents many of the policies.

Reply:
> How does it differ from Nixpkgs manual, then?

Besides https://github.com/NixOS/rfcs/pull/72 (:sob:), I think Nixpkgs is too busy to be a place to talk about policies, not necessarily closely related to the content of Nixpkgs itself.

Reply:
> How does it differ from Nixpkgs manual

Firstly, the scope is thought to be broader with a clear "meta"-focus on organization and community - _not_ code. Maybe that's becoming clearer by the measure people submit ideas for the brainstorming section. That being said, they naturally overlap. One outlook _can_ be: Nixpkgs' manual is becoming a manual for nixpkgs (and probably nix language), but policies and community organization is migrated to this repo over time.

Should I add this to the brainstorming section on your behalf? "Move non-code policies from nixpkgs/manual"

Reply:
@8573
> Discourse inferior on these points

I find it hard to pin those aspects down without illegitimately narrowing down their meaning. Maybe you can help me scoping and formulating the exemplifications:

- Strucutre: Discourse are ephemeral chats on the metropolis. There is no inherent _structure_ that transforms those conversations into directed action. There is not even a bias towards it, i would say, there is even a systemic bias towards _not_ taking action. 

- Consolidation: in conversational style, a _conclusion_ lexically is nothing more than another agent speaking: we cannot mark or systematically establish conclusions

- Reference: This combines structure and legitimacy of conclusions. A legitimate conclusion is _apt_ to act as an established reference. Structure makes them discoverable (so they are actually used as reference in every day life)

I'm giving this a shot in a commit, please help with the wording :wink: